package.json

```
@@ -12,33 +13,34 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@primer/octicons-react": "17.9.0",
-    "@primer/react": "35.15.1",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "react-query": "3.39.2",
-    "react-router-dom": "6.4.4",
-    "styled-components": "5.3.6"
+    "@primer/octicons-react": "19.16.0",
+    "@primer/react": "37.31.0",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
+    "@tanstack/react-query": "^5.85.9",
```

Remove carrot for pinned versions to align with the rest of the package.json.

```suggestion
    "@tanstack/react-query": "5.85.9",
```